proto/forch_configuration.proto

```diff
@@ -179,7 +179,7 @@ message ProxyServerConfig {
   // port number for proxy server
   int32 proxy_port = 1;
 
-  // map of targets
+  // map of targets with target name as key
```

Comment:
"target with target name" is a bit of a circular definition. It's a path prefix as key, right?

Reply:
Addressed.